addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1568,11 +1568,19 @@ var SnippetsMenu = Widget.extend({
     _activateInsertionZones: function ($selectorSiblings, $selectorChildren) {
         var self = this;
 
-        // If a modal is open, the drop zones must be created only in this modal
-        const $openModal = self.getEditableArea().find('.modal:visible');
-        if ($openModal.length) {
-            $selectorSiblings = $openModal.find($selectorSiblings);
-            $selectorChildren = $openModal.find($selectorChildren);
+        // If a modal or a dropdown is open, the drop zones must be created
+        // only in this element.
+        const $editableArea = self.getEditableArea();
+        let $open = $editableArea.find('.modal:visible');
+        if (!$open.length) {
+            $open = $editableArea.filter('.dropdown-menu.show').parent();
+            if (!$open.length) {
+                $open = $editableArea.find('.dropdown-menu.show').parent();
+            }
+        }
+        if ($open.length) {
+            $selectorSiblings = $open.find($selectorSiblings);
+            $selectorChildren = $open.find($selectorChildren);
```

Comment:
This commit should be done in a separate PR in 13.0 where the bug already occurs

Reply:
The fix is now targeting 13.0 here: https://github.com/odoo/odoo/pull/80603